module/Motion/Rotation.h

```diff
@@ -37,20 +39,27 @@ class Rotation : public Motion {
    * @brief 回頭する際の継続条件判定をする　返り値がfalseでモーターが止まる
    * @param initLeftMileage   回頭を始めた時点での左車輪の走行距離
    * @param initRightMileage  回頭を始めた時点での右車輪の走行距離
+   * @param leftSign 左車輪の回転方向
+   * @param rightSign 右車輪の回転方向
    * @note オーバーライド必須
    */
-  virtual bool isMetPostcondition(double initLeftMileage, double initRightMileage);
+  virtual bool isMetPostcondition(double initLeftMileage, double initRightMileage, int leftSign,
+                                  int rightSign)
+      = 0;
 
   /**
    * @brief 実行のログを取る
    */
-  void logRunning();
+  virtual void logRunning();
 
  private:
-  int targetValue;          // 回転角度(deg) 0~360 もしくは 指定色
-  int targetSpeed;          // 目標速度
+  // class Timer Timer;
+
+ protected:
+  double targetSpeed;       // 目標速度
```

Comment:
気づくのがすごく遅くなったけど、↓の方がいいかも

```suggestion
  SpeedCalculator speedCalculator;
```

Reply:
この場合って、事前条件判定のtargetSpeedはどういうふうに変更するのが適切なのでしょうか？

Reply:
ごめん。事前条件の判定でtargetSpeed使ってたね。こっちは戻してください。